.github/scripts/json_mocks.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
i cant figure out how to get the equivalent of merge_base_commit using graphql

Reply:
Why do you want to use any of the APIs, where it should be the feature of the repository mergebot operates on?